internal/cli/release_create.go

```diff
@@ -233,12 +234,12 @@ func (c *ReleaseCreateCommand) Synopsis() string {
 
 func (c *ReleaseCreateCommand) Help() string {
 	return formatHelp(`
-Usage: waypoint release [options] [id]
+Usage: waypoint release [options]
 
   Open a deployment to traffic.
 
   This defaults to the latest deployment. Other deployments can be
-  specified by sequence number or long ID.
+  specified by ID.
```

Comment:
```suggestion
  specified by ID using the '-deployment' flag.
```

Might be worth being explicit for how to do this